.github/workflows/rworkflows.yml

```diff
@@ -0,0 +1,56 @@
+name: rworkflows
+'on':
+  push:
+    branches:
+    - master
+    - main
+    - rworkflow
+  pull_request:
+    branches:
+    - master
+    - main
+    - rworkflow
+
+jobs:
+  rworkflows:
+    permissions: write-all
+    runs-on: ${{ matrix.config.os }}
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    container: ${{ matrix.config.cont }}
+    strategy:
+      fail-fast: ${{ false }}
+      matrix:
+        config:
+        - os: ubuntu-latest
+          bioc: devel
+          r: auto
+          cont: ghcr.io/bioconductor/bioconductor_docker:devel
+          rspm: ~
+        - os: macOS-latest
+          bioc: devel
+          r: auto
+          cont: ~
+          rspm: ~
+        - os: windows-latest
+          bioc: devel
+          r: auto
+          cont: ~
+          rspm: ~
+    steps:
+    - uses: neurogenomics/rworkflows@master
+      with:
+        run_bioccheck: ${{ false }}
+        run_rcmdcheck: ${{ true }}
+        as_cran: ${{ true }}
+        run_vignettes: ${{ true }}
+        has_testthat: ${{ true }}
+        run_covr: ${{ true }}
+        run_pkgdown: ${{ true }}
```

Comment:
This part of the action will duplicate the functionality of this workflow:
* https://github.com/awasyn/MolEvolvR/blob/rworkflow/.github/workflows/pkgdown.yaml

Reply:
You are right. I can turn off the function for now and if the need arises we switch it on then.